firefox_puppeteer/base.py

```diff
@@ -28,16 +28,44 @@ def get_marionette(self):
 class UIBaseLib(BaseLib):
     """A base class for all UI element wrapper classes inside a chrome window."""
 
-    def __init__(self, marionette_getter, window):
-        BaseLib.__init__(self, marionette_getter)
-
+    def __init__(self, marionette_getter, parent, element):
```

Comment:
So what now? I don't see a way that we could or even should duplicate the DOM regarding the parent element. We have our own class structure the test creators are working with. They do not work with the DOM.